.github/workflows/rust.yml

```diff
@@ -203,12 +201,14 @@ jobs:
 
       - name: Run tests
         working-directory: ${{ matrix.directory }}
-        run: cargo +${{ matrix.toolchain }} make --profile ${{ matrix.profile }} test --no-fail-fast
+        run: just test --no-fail-fast
 
       - name: Run miri
         if: ${{ startsWith(matrix.toolchain, 'nightly') }}
         working-directory: ${{ matrix.directory }}
-        run: cargo +${{ matrix.toolchain }} make --profile ${{ matrix.profile }} miri --no-fail-fast
+        run: |
+          rustup component add miri
```

Comment:
Not suggesting it's a problem, just seems strange that we have a manual `component add` command here where in other places we use the `components:` thing (although I appreciate that's removed in a follow-up PR)

Reply:
`miri` is special here as it's never available in non-nightly toolchains.

Reply:
Understood, thanks for clarifying :)